recipes/openssl/3.x.x/conanfile.py

```diff
@@ -136,6 +140,8 @@ def layout(self):
     def requirements(self):
         if not self.options.no_zlib:
             self.requires("zlib/[>=1.2.11 <2]")
+        if self.options.use_validated_fips:
+            self.requires("openssl/3.0.9", visible=False, libs=False, headers=False, run=False)
```

Comment:
this version should probably added again to config.yml and conandata.yml

Reply:
Good call, added it back in [3c52995](https://github.com/conan-io/conan-center-index/pull/24709/commits/3c5299580b24848f22b4da856e4ea3ed38ec9210)